src/tests/functional/plugin/shared/include/behavior/infer_request/config.hpp

```diff
@@ -117,7 +117,7 @@ TEST_P(InferRequestConfigTest, ReusableCPUStreamsExecutor) {
         // Load CNNNetwork to target plugins
         execNet = ie->LoadNetwork(cnnNet, target_device, config);
         execNet.CreateInferRequest();
-        if (target_device == ov::test::utils::DEVICE_KEEMBAY) {
```

Comment:
General comment, please do not hardcode such constructions in the shared test lib. TODO: remove